src/core/thread/mesh_forwarder_ftd.cpp

```diff
@@ -87,14 +88,35 @@ void MeshForwarder::SendMessage(OwnedPtr<Message> aMessagePtr)
                         mIndirectSender.AddMessageForSleepyChild(message, child);
                     }
                 }
+
+#if OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_ENABLE
+                if (destinedForAll)
```

Comment:
followup: I guess there's a question if we should forward link and realm-local addresses to RX off when idle enh CSL neighbors.

Reply:
nevermind this, I didn't realize this code is for all nodes, including sleepy ones.